seed/challenges/02-javascript-algorithms-and-data-structures/functional-programming.json

```diff
@@ -289,7 +289,7 @@
       ],
       "tests": [
           "assert(fixedValue === 4, 'message: Your function <code>incrementer</code> should not change the value of <code>fixedValue</code>.');",
-          "assert(code.match(/function\\s+?incrementer\\(.+?\\)/g), 'message: Your <code>incrementer</code> function should take a parameter.');",
+          "assert(code.match(/function\\s+?incrementer\\s*\\(.+?\\)/g), 'message: Your <code>incrementer</code> function should take a parameter.');",
```

Comment:
Can you make this a lazy match please with the `*?` quantifier

i.e. `incrementer\\s*?\\(`